### ISSUE TITLE
I don't think DDP is getting distributed?

### ISSUE THREAD

**Issue by lorensr** [meteorhacks/cluster#78]
latest versions of both `cluster` and `mup`
- `Meteor.default_connection._stream.socket._base_url` it is always `"https://myapp.io/cluster-ddp/963597a14face091bba98e1727674a0866878bcc/web"`, even after switching browsers or clearing cookies. 
- When I `console.log 'here'` in `Meteor.publish`, `mup logs` always prints out `[ip1] here`, even after switching browsers or clearing cookies. 
 
Is something wrong with my `mup.json`?
```javascript
{
  // Server authentication info
  "servers": [
    {
      "host": "ip1"
      "username": ...
      "pem": ...
    },
    {
      "host": "ip2"
      "username": ...
      "pem": ...
    },
    {
      "host": "ip3"
      "username": ...
      "pem": ...
    }
  ],

  "setupMongo": false,
  "setupNode": true,
  "nodeVersion": "0.10.36",
  "setupPhantom": true,
  "appName": "myapp",

  // Location of app (local directory)
  "app": "~/myapp",

  "ssl": {
    "pem": "./ssl.pem"
  },

  "env": {
    "ROOT_URL": "https://myapp.io",
    "MONGO_URL": ...
    "MONGO_OPLOG_URL": ...
    "CLUSTER_DISCOVERY_URL": ...
    "CLUSTER_SERVICE": "web"
  },

  // Meteor Up checks if the app comes online just after the deployment
  // before mup checks that, it will wait for no. of seconds configured below
  "deployCheckWaitTime": 15
}
```

**Comment by lorensr** [comment]
Are there better ways to debug whether connections are getting distributed? And can you add this to the readme?

**Comment by lorensr** [comment]
I upgraded Kadira to show hosts. The one getting most of the sessions is `ip1`, but it looks like a few might be going to the others – why wouldn't it be evenly distributed?
![image](https://cloud.githubusercontent.com/assets/251288/8864821/bbefa324-3174-11e5-8d7b-64d46a9336b7.png)

**Comment by elie222** [comment]
I just ran into a situation where one of my cores was running at 100% CPU and the other at 18%. (Using multicore support for 2 cores)

**Comment by elie222** [comment]
Any answers here? I want to move to multi server setup but not good if distribution isn't working properly.

**Comment by elie222** [comment]
@lorensr what happens if you explicitly put the weight for each of the three servers to 0.33?

**Comment by elie222** [comment]
Sorry. Doesn't matter. Wouldn't make sense. Each server would just pass on 2 out of 3 requests to other servers if you did that

**Comment by lorensr** [comment]
Using `meteor-down`, I'm seeing sessions in Kadira get more evenly distributed. CPU on the A-record machine quickly gets to 100%  even at concurrency level `1` (70% stud and 30% meteorup). The other machines stay at 10% CPU even at concurrency `100`. How much would this imbalance be helped by making some of the other machines balancers? Or do I need to switch to a DNS with round-robin A records to make a big difference? Do balancers do some of the stud work? /cc @arunoda

**Comment by elie222** [comment]
I've been seeing the same issues for a 2 core machine. With one process running at 100% CPU and the other at 2% CPU. I'm using nginx as a reverse proxy with ip_hash for balancing across multiple servers for the moment.

**Comment by elie222** [comment]
@lorensr I just noticed, shouldn't you be exporting the endpointUrl as an environment variable?

**Comment by lorensr** [comment]
Looks from the example like it's not needed with `mup`:
https://github.com/meteorhacks/cluster#practical-setup
https://gist.github.com/arunoda/13f2e9c22bf526b84556

**Comment by elie222** [comment]
Yeah... Mup takes care of it. Just remembered. But what about balancer url?

**Comment by lorensr** [comment]
No balancers – hence question on whether adding balancers would fix vs rotating A record.

**Comment by elie222** [comment]
Well try it. You don't need to do anything with a records to get the
balancing working

**Comment by lorensr** [comment]
SSL didn't cover subdomains, and then I had to remove cluster to get hot code reload working.

**Comment by maxkferg** [comment]
This is normally a problem with the system clock. If the servers' clocks are out of sync by more than 10 seconds, the balancing algorithm breaks down. See #105 for details
